### PR TITLE
Amos.ux tweaks

### PR DESCRIPTION
**Pull request by amenders**

### CODE REVIEW

File: app/templates/info.hbs [hunk 1]
```diff
@@ -13,8 +11,10 @@
         Contact: <br> 
         {{model.tool.phonenumber}} <br>
         {{model.tool.email}}<br><br>
+		Year: {{model.tool.year}} <br>
 		Purchase date: {{model.toolpurchasedate}}<br>
```
Comment (Jbcampbe) [inline app/templates/info.hbs]: looks like this is missing a `.` in between `tool` and `purchase`

**Comment by Jbcampbe** [comment]
The `back` button keeps appearing in different spots. If I keep clicking some random tool and then going back to the search page, switch to the transfer page and click a random tool there repeatedly the back button on the info page sometimes shows up at the top and sometimes at the botom

**Comment by ekraihan** [comment]
Amo, when you clear the search filters, i wondered about switching them to "Status" or "Brand" vs. "Any Status" or "Any Brand"

Also, i wondered how easy/necessary it would be to implement clearing the search input field when you click a filter :)

**Comment by amenders** [comment]
fixed period, fixed clearing search bar. I don't think I can do much about back button placement: I'm not thinking that it's me...

**Pull request merged**